src/hotspot/share/opto/type.hpp

```diff
@@ -1473,6 +1473,7 @@ class TypeAryPtr : public TypeOopPtr {
   virtual const TypeKlassPtr* as_klass_type(bool try_for_exact = false) const;
 
   // Convenience common pre-built types.
+  static const TypeAryPtr* BOTTOM;
```

Comment:
While you are here it may be better to change the other constant to `TypeAryPtr*` instead of `TypeAryPtr *`

Reply:
Right, done.